habitat_baselines/utils/common.py

```diff
@@ -180,9 +180,12 @@ def get(
             and device.type == "cuda"
             and cache.device.type == "cpu"
         ):
+            cache = cache.pin_memory()
```

Comment:
Ping @erikwijmans about this part.

Reply:
Oops I have no idea how that got there, thanks for catching it, this is now removed.